tests/rptest/tests/acls_test.py

```diff
@@ -245,6 +245,38 @@ def check_super_user_perms():
                            timeout_sec=timeout_sec,
                            err_msg=f'super user: {err_msg}')
 
+    @cluster(num_nodes=1)
```

Comment:
fix: should be 3... looks like the test is failing because you don't have enough nodes

Reply:
fair... either way this is why the test is failing